docs/workloads/azure/data/data_engineering/data_quality_azure.md

```diff
@@ -120,3 +120,22 @@ Here's a minimal example of a configuration file:
     ]
 }
 ```
+
+## Validation Results
+
+Results of the data quality checks are stored in the `dq_output_path` location in Delta tables. The results contain the following columns:
```

Comment:
Mention the naming of the tables (e.g. the same as the input table with `_dq` appended).